docs: add k8s guide and Llama-3.2-3b-Instruct example-run (#161)
neuralmagic/guidellm: docs/guides/k8s/README.md

> ## Run Guidellm with Kubernetes Job

Reviewer: There are a bunch case issues with the GuideLLM name. Please fix to be constant with the rest of our documentation.

```suggestion
## Run GuideLLM with Kubernetes Job
```

Here's an example to run `guidellm` with `meta-llama/Llama-3.2-3B-Instruct` that has been deployed with
[llm-d-deployer](https://github.com/neuralmagic/llm-d-deployer/blob/main/quickstart/README-minikube.md).
Replace the `--target` and references to `Llama-3.2-3B` in [guidellm-job.yaml](./guidellm-job.yaml) to evaluate any served LLM.

### Run evaluation

```bash
# Update the claim-name in accessor-pod.yaml, and guidellm-job.yaml if using a different pvc-name
kubectl apply -f pvc.yaml
kubectl apply -f guidellm-job.yaml
```

> > **📝 NOTE:** [Dockerfile](./Dockerfile) was used to build the image for the guidellm-job pod.
> 

Reviewer: ```suggestion
```

> **📝 NOTE:** The HF_TOKEN is passed to the job, but this will not be necessary if you use the same PVC as the one storing your model.
> Guidellm uses the model's tokenizer/processor files in its evaluation. You can pass a path instead with `--tokenizer=/path/to/model`.

> > This eliminates the need for Guidellm to download the files from Huggingface.

Reviewer: ```suggestion
> This eliminates the need for GuideLLM to download the files from Hugging Face.
```
